src/components/plugin/plugin-app.tsx

```diff
@@ -15,6 +15,11 @@ import * as css from "./plugin-app.scss";
 import * as icons from "../common/icons.scss";
 import { POEDITOR_LANG_NAME } from "../../utils/poeditor-language-list";
 
+export interface IPluginEvent {
+  type: string;
+  text: string;
+}
```

Comment:
I'd be inclined to include a `bounds` element, e.g.
```typescript
  bounds?: DOMRect;
```
for use when we decide to solve the positioning problem. Then the client just needs to call `getBoundingClientRect()` on the element to get the bounds, i.e. the `bounds` element is assumed to be relative to the viewport.

Reply:
I'm going to add the bounds for now, we can use it for future positioning if needed.